code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -270,7 +270,7 @@
 	caliber = "a556"
 	matter = list(DEFAULT_WALL_MATERIAL = 1800)
 	ammo_type = /obj/item/ammo_casing/a556
-	max_ammo = 10
+	max_ammo = 30

```

Comment:
The bulldog has an underbarrel grenade launcher and fires a different round. It's not just an STS resprite.